benches/WorkerManager.ts

```diff
@@ -24,7 +25,7 @@ async function main() {
   // 1 KiB of data is still too small
   const bytes = crypto.randomBytes(1024 * 1024);
   const summary = await b.suite(
-    'WorkerManager',
+    path.basename(__filename, path.extname(__filename)),
     b.add('Call Overhead', async () => {
       // This calls a noop, this will show the overhead costs
       // All parallelised operation can never be faster than this
```

Comment:
You need `utils/utils.ts` and `utils/index.ts`. They contain the common suite. Then you just replace the bottom `cycle` and `complete` and `save` with the `...suiteCommon`.